### ISSUE TITLE
My list

### ISSUE THREAD

**Issue by mulundapm** [bcc-code/brunstadtv#325]
**User story**
As a user I want save episode to a section so that I can easily trace back to these previously saved videos

**Acceptance criteria**
1. Able to have "My list" as a section and able to add this to any page
2. Create a section type which retrieves the items from my list
3. User able to add/ remove items from my list
4. The "my list" section should be refreshed when the user adds/removes items

Design shows my list as a tab but would want to have it as a section in pages before we have more data on how popular it is
[Figma design](https://www.figma.com/file/BMXWzaA93qm8jXKjyPiFOs/BCC-Media-%E2%80%93-Mobile-App-V4?node-id=8822%3A51965&t=m5T6EAvQ6lmdOZPP-1)

**Comment by fredrikved** [comment]
Not sure if it makes sense to support easily adding/removing things from the collection, but it should be doable to create a section type which retrieves the items from my list, in the same way as continue watching.

**Comment by andreasgan** [comment]
Eliezer has fully implemented it as a page now @mulundapm, I completely forgot that you mentioned to me that we should try it as a section first..
What should we do?

**Comment by mulundapm** [comment]
@fredrikved Yes was thinking to have it same way as continue watching. Have edited the task description.

**Comment by mulundapm** [comment]
@andreasgan Does it only require change from Eliezer side? If yes, unfortunately will have to remove it. My apology for not writing it down somewhere.

**Comment by andreasgan** [comment]
done via https://github.com/bcc-code/bcc-media-app/pull/143
